tests/test_oauth.py

```diff
@@ -0,0 +1,169 @@
+    location_authorize = location
+    response_authorize = requests.get(
+        location_authorize, headers={"cookie": "token=huggingface-hub.js-cookie"}, allow_redirects=False
+    )
+    assert response_authorize.status_code == 303
+    assert "location" in response_authorize.headers
+
+    # Make call to callback
+    location_callback = response_authorize.headers["location"].replace(
+        "http://localhost:3000/", "http://testserver/oauth/huggingface/callback"
+    )
+    response_callback = client.get(location_callback, headers={"cookie": session_cookie}, follow_redirects=False)
+    assert response_callback.status_code == 307
+    assert response_callback.headers["location"] == "/"
+    assert "set-cookie" in response_callback.headers
+    new_session_cookie = response_callback.headers["set-cookie"].split(";")[0]
+    assert len(session_cookie) < len(new_session_cookie)  # oauth data has been added to the cookie
+
+    # Finally make a call to the root
+    response_hello = client.get("/", headers={"cookie": new_session_cookie}, follow_redirects=False)
+    assert response_hello.status_code == 200
+    data = response_hello.json()
+    assert data["msg"] == "Hello, hub.js!"
+    assert data["oauth_info"]["access_token"] is not None
+    assert data["oauth_info"]["scope"] == "openid profile email"
+    assert data["oauth_info"]["user_info"] == {
+        "sub": "62f264b9f3c90f4b6514a269",
+        "name": "@huggingface/hub CI bot",
+        "preferred_username": "hub.js",
+        "email_verified": True,
+        "email": "eliott@huggingface.co",
+        "picture": "https://hub-ci.huggingface.co/avatars/934b830e9fdaa879487852f79eef7165.svg",
+        "profile": "https://hub-ci.huggingface.co/hub.js",
+        "website": "https://github.com/huggingface/hub.js",
+        "is_pro": None,
+        "can_pay": None,
+        "orgs": None,
+    }
+
+
+def test_get_oauth_uris_default():
+    login_uri, callback_uri, logout_uri = _get_oauth_uris()
+    assert login_uri == "/oauth/huggingface/login"
+    assert callback_uri == "/oauth/huggingface/callback"
+    assert logout_uri == "/oauth/huggingface/logout"
+
+
+def test_get_oauth_uris_with_prefix_stripped():
+    login_uri, callback_uri, logout_uri = _get_oauth_uris("my/custom/router")
+    assert login_uri == "/my/custom/router/oauth/huggingface/login"
+    assert callback_uri == "/my/custom/router/oauth/huggingface/callback"
+    assert logout_uri == "/my/custom/router/oauth/huggingface/logout"
+
+
+def test_get_oauth_uris_with_prefix_not_stripped():
+    login_uri, callback_uri, logout_uri = _get_oauth_uris("/my/custom/router/")
+    assert login_uri == "/my/custom/router/oauth/huggingface/login"
+    assert callback_uri == "/my/custom/router/oauth/huggingface/callback"
+    assert logout_uri == "/my/custom/router/oauth/huggingface/logout"
+
+
+def test_get_mocked_oauth_info():
+    oauth_info = _get_mocked_oauth_info()
```

Comment:
```suggestion
def test_get_mocked_oauth_info(monkeypatch):
    monkeypatch.setenv("HF_TOKEN", TOKEN)
    oauth_info = _get_mocked_oauth_info()
 
```
This should fix the failing test. Alternatively, one can patch the `get_token()` function